tools/tslint-rules/noExposedTodoRule.js

```diff
@@ -0,0 +1,32 @@
+const ts = require('typescript');
+const utils = require('tsutils');
+const Lint = require('tslint');
+
+const ERROR_MESSAGE = 'Todo inside of the comment might be published in the docs.';
```

Comment:
```
A TODO may only appear in inline (//) style comments. This is meant to prevent
a TODO from beig accidentally included in any public API docs.
```